frontend/web/src/contexts/WebSocketContext.tsx

```
@@ -0,0 +1,188 @@
+
+      case 'model_update':
+        const modelUpdate = message.payload as ModelUpdate;
+        setModelStatus((prev) => {
+          const updated = new Map(prev);
+          updated.set(modelUpdate.model_id, modelUpdate);
+          return updated;
+        });
+        break;
+
+      case 'system_message':
+        const sysMsg = message.payload as SystemMessage;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)